src/importer.cpp

```diff
@@ -193,21 +195,21 @@ bool Importer::ImporterImpl::checkComponentForCycles(const ComponentPtr &compone
     if (component->isImport()) {
         auto model = component->importSource()->model();
         if (model == nullptr) {
-            auto issue = Issue::create();
-            issue->setDescription("Component '" + component->name() + "' requires a model imported from '" + resolvingUrl + "' which is not available in the importer.");
-            issue->setLevel(Issue::Level::ERROR);
-            issue->setImportSource(component->importSource());
-            issue->setReferenceRule(Issue::ReferenceRule::IMPORTER_NULL_MODEL);
+            auto issue = Issue::IssueImpl::create();
+            issue->mPimpl->setDescription("Component '" + component->name() + "' requires a model imported from '" + resolvingUrl + "' which is not available in the importer.");
+            issue->mPimpl->setLevel(Issue::Level::ERROR);
```

Comment:
Redundant.

Reply:
Should have chcked the whole thing first. Ok, fixed this one and other similar ones in commit 7a25f5ed.